io.home-assistant.media-player/manifest.tt

```diff
@@ -0,0 +1,43 @@
+abstract class @io.home-assistant.media-player
+#_[thingpedia_name="Media Player"]
+#_[thingpedia_description="Interface for Home Assistant's Media Player."]
+#_[description="Interface for Home Assistant's Media Player."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(on,off))
+  #_[canonical="switch state"]
+  #_[confirmation="the state of your media player"]
+  #_[formatted=["Your media player is ${state}"]];
+
+  action turn_on()
+  #_[canonical="turn on media player"]
+  #_[confirmation="turn on your media player"]
+  #[confirm=true];
+
+  action turn_off()
+  #_[canonical="turn off media player"]
+  #_[confirmation="turn off your media player"]
+  #[confirm=true];
+
+  action volume_up()
+  #_[canonical="increase media player volume"]
+  #_[confirmation="increase your media player's volume"]
+  #[confirm=true];
+
+  action volume_down()
+  #_[canonical="decrease media player volume"]
+  #_[confirmation="decrease your media player's volume"]
+  #[confirm=true];
+
+  action volume_mute()
+  #_[canonical="mute media player"]
+  #_[confirmation="mute your media player"]
+  #[confirm=true];
+
+  action volume_unmute()
+  #_[canonical="unmute media player"]
+  #_[confirmation="unmute your media player"]
+  #[confirm=true];
```

Comment:
Is there a set_volume(percent) function in Home Assistant?

Reply:
Yup there's a volume_set, I'll include that in